core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -544,11 +544,12 @@ impl<T: LikeClusterInfo> SchedulerController<T> {
                     .is_ok()
                 })
                 .filter_map(|(packet, tx, deactivation_slot)| {
-                    process_compute_budget_instructions(SVMMessage::program_instructions_iter(&tx))
-                        .map(|compute_budget| {
-                            (packet, tx, deactivation_slot, compute_budget.into())
-                        })
-                        .ok()
+                    process_compute_budget_instructions(
+                        SVMMessage::program_instructions_iter(&tx),
+                        &working_bank.feature_set,
```

Comment:
Just want to call out that I think that root bank and working bank are both fine here because if we end up processing the transaction in a different epoch than when we sanitized it, the fees could be different but we recalculate them anyways.